calico_rkt/ipam.py

```diff
@@ -0,0 +1,173 @@
+# Copyright 2015 Metaswitch Networks
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import json
+import os
+import sys
+
+from pycalico.ipam import IPAMClient
+from util import configure_logging
+
+LOG_FILENAME = "ipam.log"
+
+_log = logging.getLogger(__name__)
+datastore_client = IPAMClient()
+
+# Error codes
+ERR_CODE_UNHANDLED = 100
```

Comment:
These should probably be common so we don't duplicate error codes (though I admit I don't think we need error codes just yet, can probably use the same code for all errors)